### PR TITLE
Distinguish between failed and timed out in script run

### PR DESCRIPTION
**Pull request by okurz**

### CODE REVIEW

**Comment by okurz** [comment]
@asmorodskyi

File: testapi.pm [hunk 1]
```diff
@@ -726,6 +726,14 @@ sub x11_start_program {
     return $distri->x11_start_program($program, $timeout, $options);
 }
 
+sub _handle_script_run_ret {
+    my ($ret, $cmd, %args) = @_;
```
Comment (asmorodskyi) [inline testapi.pm]: can we get even more info from $ret value ? like what was error code at least or something like this

**Comment by okurz** [comment]
tests fixed

File: testapi.pm (hunk 1 above)
Comment (okurz) [inline testapi.pm]: the $ret value is visible in the info thumbnails within openQA already showing what the script returned.

**Pull request merged**